elasticdl/pkg/common/tensor.go

```diff
@@ -0,0 +1,54 @@
+package common
+
+import (
+	"bytes"
+	"elasticdl.org/elasticdl/pkg/proto"
+	"encoding/binary"
+	"math"
+)
+
+// Tensor defines tensor struct
+// TODO(qijun): handle different tensor dtype
+type Tensor struct {
+	Name    string
+	Value   []float32
+	Dim     []int64
+	Indices []int64
+}
+
+// GetTensorSize returns the size of a tensor
+func GetTensorSize(t *Tensor) int64 {
+	var size int64 = 1
+	for _, d := range t.Dim {
+		size *= d
+	}
+	return size
+}
+
+// DeserializeTensorPB transforms pb to tensor
+func DeserializeTensorPB(pb *proto.Tensor, t *Tensor) {
```

Comment:
What is the usage of this func? Should `Tensor` be pre-defined? Or we can define a func with a input as `*proto.Tensor` and returns a `*Tensor`?

Reply:
Done